common/web/lm-worker/src/main/model-compositor.ts

```diff
@@ -434,18 +434,24 @@ export default class ModelCompositor {
     });
 
     let suggestions = suggestionDistribution.splice(0, ModelCompositor.MAX_SUGGESTIONS).map(function(value) {
-      if(value.sample['p']) {
+      let sample: Suggestion & {
+        p?: number,
+        "lexical-p"?: number,
+        "correction-p"?: number
+      } = value.sample;
```

Comment:
Where else would this type be used?

Reply:
These fields are only used for debugging; I've found them helpful in the past when determining what influenced the suggestions being displayed and how strongly earlier suggestions "won" over later ones.  These pass through the lm-layer -> Web-engine boundary and are available for the `Suggestion` objects held by the suggestion banner, allowing them to be viewable without needing any breakpoints.  

Without these two fields, the values they represent are locked within a limited time-window within the lm-worker, requiring breakpoints at the right location.

Of note:  `p`, at least, would be _very_ useful for implementing autocorrect.  There's a chance one of the other two would be relevant as well.  For example, we may wish to require much stricter keystroke input ("correction") for rarer words, possibly raising the overall threshold required to trigger for such cases.

Reply:
The closest related issue I could find re: autocorrect: #1841.  There doesn't seem to be a baseline "autocorrect" issue, but #1841 at least references the notion of autocorrect.